New introduction page

vvandijck opened the pull request:
Created an intro page when starting up the project. It gives an overview about the project and some interesting links for further reading.

Commit:
feat(intro): Added an intro page to the project. (#2)

Commit:
fix(build-css) single quotes.

Commit:
fix(build-css) removed redundant comma.

jbmoelker commented on src/intro.html:
```diff
@@ -0,0 +1,68 @@
+            <p>Any additions to these lists are welcome.</p>
+            <h3>Good:</h3>
+            <ul>
+                <li><a href="http://www.nomensa.com/">Nomensa</a> - Good contrast</li>
```
none of these examples work in the a11y-viewer :-/

Commit:
Created separate assets build task.

Added assets build task. Renamed files to index.

Commit:
Improved introduction page.

Added more facts and useful information about accessibility. Improved
styling and layout.
